netci.groovy

```diff
@@ -1442,6 +1442,11 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${arch} ${buildOpts}"
                     }
 
+                    // If it is a release build for windows, ensure PGO is used, else fail the build
```

Comment:
Let me know if this commit doesn't fix that 8cff14e28ebf2aaa31956962231a54bdab1974a6